source/extensions/filters/common/lua/lua.h

```diff
@@ -69,6 +69,15 @@ namespace Lua {
   lua_pushnumber(state, val);                                                                      \
   lua_settable(state, -3);
 
+/**
+ * Check string from lua with length
+ */
+inline absl::string_view checkLuaStringWithLength(lua_State* state, int arg) {
```

Comment:
`getStringViewFromLuaString()`? And adding `doxygen`-like comment will be really appreciated. Something like:

```
/**
 * Get absl::string_view from Lua string. This checks if the argument at index is a string
 * And build a nabsl::string_view from it.   
 * @param state the current Lua state.
 * @param index the index of argument.
 * @return absl::string_view of Lua string with proper string length.
 **/
```